examples/grafana-integration/README.md

```diff
@@ -0,0 +1,26 @@
+# Hot R.O.D. - Rides on Demand  - Grafana integration
+
+This is the Hot R.O.D. demo application that consists of the same components as the `examples/hotrod/`, only Grafana and Loki integration is added to this setup, so you can correlate logs and traces in one application.
+
+## Running
+
+### Run everything via `docker-compose`
+
+#### Prerequisites
+* Download `docker-compose.yml` from https://github.com/jaegertracing/jaeger/blob/master/examples/grafana-integration/docker-compose.yml
+* Download the `datasources.yaml` from the `examples/grafana-integration/` folder
```

Comment:
Considering that there are several files in play here, I would rather recommend that people clone the repository and `cd examples/grafana-integration`.